installation.md

```diff
@@ -17,10 +17,18 @@ After installing, `clangd --version` should print `clangd version 7.0.0` or late
 
 <details>
 <summary markdown="span">Mac OS X</summary>
+ 
 Clangd can be installed (along with LLVM) via [Homebrew](https://brew.sh):
 ```
 brew install llvm
 ```
+
+or with [MacPorts](https://www.macports.org/):
+
+```
+sudo port install clang-11
```

Comment:
Shouldn't this be clangd-11

Reply:
clangd is installed as part of the clang package.

Reply:
It's a bit unfortunate this has a version number in it, but I'm sure people can work that out if our docs get stale.

(Versioning compiler packages makes sense of course, clangd less so but bundling them makes practical sense too, so...)

Reply:
Would it make sense to change it to something like this instead:

````markdown

or with [MacPorts](https://www.macports.org/) using one of the available `clang` ports, e.g.

```
sudo port install clang-11
```

Use `port search clang` to find all available versions.
````

In this case `clang-11` is just an example, so even when `clang-12` or `clang-13` are released, the docs won't appear stale.